Frontend/Frontend.Client/Models/WebSocketService.cs

```diff
@@ -29,35 +35,37 @@ public async Task ConnectAsync(string uri)
 
     private void OnMessage(string message)
     {
-
         try
         {
-            var lines = message.Split(new[] { "\r\n", "\n" }, StringSplitOptions.None);
-
-            if (lines.Length >= 6 &&
-                lines[0].StartsWith("DO SEND") &&
-                lines[1].StartsWith("FROM ") &&
-                lines[2].StartsWith("IN general-chat") &&
-                lines[3].StartsWith("AT ") &&
-                lines[4].StartsWith("WITH"))
+            if (message.StartsWith("DO ACCEPT"))
```

Comment:
We send DO ACCEPT as a response for many things, you'll need to specify this DO ACCEPT behind another condition ensuring its the DO ACCEPT related to the AUTH request